packer/do-packer.json.pkr.hcl

```diff
@@ -26,8 +27,13 @@ variable "region" {
   description = "The name (or slug) of the region to launch the droplet in."
 }
 
-# "timestamp" template function replacement
-locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
+variable "tag" {
+  type        = string
+  default     = ""
+  description = "The image tag"
+}
+
+locals { tag = substr("${var.tag}", 0, 7) }
```

Comment:
Our tags would comply with [Semver](https://semver.org/) (e.g. 1.2.3) so we shouldn't enforce taking a substring out of it